src/commands.js

```diff
@@ -155,7 +158,8 @@ function addCommandsToHelp() {
 	addToHelp('wizzy summarize ENTITY ENTITY_NAME', 'summarize a large entity in a short user-friendly manner.');
 	addToHelp('wizzy upload s3 ENTITY ENTITY_NAME', 'upload entities to S3.');
 	addToHelp('wizzy download s3 ENTITY ENTITY_NAME', 'download entities from S3.');
-
+	addToHelp('wizzy alerts pause', 'pause all the alerts in grafana');
+	
```

Comment:
can you please change this to "wizzy pause alerts"? I think this is more intuitive.